app/models/peer_review/challenge.rb

```diff
@@ -4,19 +4,13 @@ class PeerReview::Challenge < ApplicationRecord
 
   validates_presence_of :title, :difficulty, :instructions,
                         :reviewer_instructions
-  validate :difficulty_range
+  validates_numericality_of :difficulty, greater_than_or_equal_to: 1, less_than_or_equal_to: 5, message: "cannot be less than 1 or greater than 5"
```

Comment:
Un último cambio: ponele otro mensaje. "must be between 1 and 5"

Reply:
listo